packages/lib/smart-account/src/KeyStore/IKeyStore.ts

```diff
@@ -0,0 +1,28 @@
+import { SignedUserProfile } from '@dm3-org/dm3-lib-profile';
+
+//Struct that represents a Dm3 key store
+export interface Dm3KeyStore {
+    [signerAddress: string]: {
+        signerPublicKey: string;
+        encryptedProfileKeys?: string;
+    };
+}
+
+export interface IKeyStoreService {
+    //Method to publish the Dm3 user profile. Ideally this should be ENS or any other place that can be access using a resolver
```

Comment:
Using block comment style would allow the IDE to render a nice message on hover. Looks not so nice in the code, but might be useful for SDK users.

```suggestion
     /**
     * Method to publish the Dm3 user profile. Ideally this should be ENS or any other place that can be access using a resolver
     */
```
![Uploading Screenshot 2024-11-12 at 19.04.07.png…]()